src/transformers/models/auto/tokenization_auto.py

```diff
@@ -356,6 +358,7 @@
                 "LlamaTokenizerFast" if is_tokenizers_available() else None,
             ),
         ),
+        ("mistral-common", ("MistralCommonTokenizer" if is_mistral_common_available() else None, None)),
```

Comment:
```suggestion
```
what you want to do is rather update the mapping for all models released by mistral, to use `MistralCommonTokenizer` if available else what was previously there